### PR TITLE
Remove most of the material on in-house reuse from the Introduction

### PR DESCRIPTION
**Pull request by ethanwhite**
My attempt at reducing the amount of discussion of personal and in house benefits in the Introduction to make it flow better. This includes moving that material to the Conclusions.

### CODE REVIEW

**Comment by embaldridge** [comment]
This approach seems promising.  Points 1,2, and 3 in section one are reading a little strangely for me.  Maybe too passive in tone?

**Comment by dmcglinn** [comment]
Great job! I think the intro is great.  The three points in the conclusion are a bit wordy, here my attempt to make them more concise:
1. Data that is well documented is easier to understand
2. Data that is well formatted and structured is easier to use in a variety of different analysis programs.
3. Data that is shared in established repositories with open licenses is easy for others to find and use with confidence.

**Comment by ethanwhite** [comment]
I'm glad to hear that this looks promising. I've taken Dan's suggestions, tightened them further, and integrated them into the current PR.

P.S. You can do this sort of think very easily using the `--amend` option for `git commit`, which makes it easy to update the last commit for typos and suggestions made in PRs like this one.

**Comment by dmcglinn** [comment]
Looks good. I didn't know about the amend option, cool! Also add a period at the end of the sentence:
"1. Well documented data is easier to understand "

**Comment by sarahsupp** [comment]
It looks good to me!

**Comment by ethanwhite** [comment]
Fixed the period. It sounds like we're happy with things so someone should feel free to go ahead and merge this.

A note on `--amend`. It should only be used for changes that aren't on GitHub or are in a fork that isn't being used except to contribute to something else, or in repos that no one else is contributing to. The reason for this is that it rewrites history and can cause issues. As a result, if you're pushing an amended change to a public repo git will make sure that you want to do this by requiring a `-f` flag in the push (which stands for force).
